geoportal/c2cgeoportal_geoportal/scaffolds/create/front/haproxy.cfg.tmpl

```diff
@@ -71,6 +71,8 @@ frontend plain
     # If the path starts with /tiles/, use the tilecloudchain backend
     acl is_tiles var(req.path) -m beg ${VISIBLE_ENTRY_POINT}tiles/
     use_backend tilecloudchain if is_tiles
+    acl is_tiles var(req.path) -m beg ${VISIBLE_ENTRY_POINT}static_tiles/
+    use_backend tilecloudchain if is_tiles
 
```

Comment:
Please define the env var C2C_BASE_PATH=/c2c_tiles and add the rule to the front